tests/DanBettles/Telex/MatchTest.php

```diff
@@ -9,15 +9,16 @@
 
 use DanBettles\Telex\Match;
 use DanBettles\Telex\Candidate;
+use PHPUnit\Framework\TestCase;
 
-class MatchTest extends \PHPUnit_Framework_TestCase
+class MatchTest extends TestCase
 {
     public function testIsInstantiable()
     {
         $candidate = new Candidate('(01234) 567890');
         $match = new Match($candidate, '01234567890');
 
-        $this->assertSame($candidate, $match->getCandidate());
```

Comment:
Please restore this line.  I used `assertSame()` -- instead of just `assertInstanceOf()` -- because I wanted to ensure that the returned candidate is the exact same instance, not only the same type, that was used to create the instance of `Match`.  Thanks